static/app/views/explore/spans/tour.tsx

```diff
@@ -0,0 +1,208 @@
+import {t} from 'sentry/locale';
+import {space} from 'sentry/styles/space';
+import {useIsNavTourActive} from 'sentry/views/nav/tour/tour';
+
+export const enum ExploreSpansTour {
+  // Specify keywords to narrow down search
+  SEARCH_BAR = 'search-bar',
+  // Specify the query to list/aggregate results
+  TOOLBAR = 'toolbar',
+  // View the results of the query
+  RESULTS = 'results',
+}
+
+export const ORDERED_EXPLORE_SPANS_TOUR = [
+  ExploreSpansTour.SEARCH_BAR,
+  ExploreSpansTour.TOOLBAR,
+  ExploreSpansTour.RESULTS,
+];
+
+export const EXPLORE_SPANS_TOUR_GUIDE_KEY = 'tour.explore.spans';
+
+export const ExploreSpansTourContext =
+  createContext<TourContextType<ExploreSpansTour> | null>(null);
+
+export function useExploreSpansTour(): TourContextType<ExploreSpansTour> {
+  const tourContext = useContext(ExploreSpansTourContext);
+  if (!tourContext) {
+    throw new Error('Must be used within a TourContextProvider<ExploreSpansTour>');
+  }
+  return tourContext;
+}
+
+interface ExploreSpansTourModalProps {
+  closeModal: () => void;
+  handleDismissTour: () => void;
+  handleStartTour: () => void;
+}
+
+export function ExploreSpansTourModal({
+  closeModal,
+  handleDismissTour,
+  handleStartTour,
+}: ExploreSpansTourModalProps) {
+  return (
+    <TourContainer>
+      <ModalImage src={exploreSpansTourSvg} />
+      <TextContainer>
+        <Title>{t('How to Query')}</Title>
+        <Header>{t('Debug Like a Pro')}</Header>
+        <Description>
+          {t(
+            'Aggregate and visualize metrics with your span data in this new query builder. You’ll be able to drill into the exact problems causing your infra bills to spike and your users to grimace.'
+          )}
+        </Description>
+        <Footer>
+          <TextTourAction
+            size="sm"
+            onClick={() => {
+              handleDismissTour();
+              closeModal();
+            }}
+            borderless
+          >
+            {t('Maybe later')}
+          </TextTourAction>
+          <TourAction
+            size="sm"
+            onClick={() => {
+              handleStartTour();
+              closeModal();
+            }}
+            borderless
+            autoFocus
+          >
+            {t('Take a tour')}
+          </TourAction>
+        </Footer>
+      </TextContainer>
+    </TourContainer>
+  );
+}
+
+// Displays the introductory tour modal when a user is entering the experience for the first time.
+export function useExploreSpansTourModal() {
+  const hasOpenedTourModal = useRef(false);
+  const {isRegistered, startTour, endTour} = useExploreSpansTour();
+  const {data: assistantData} = useAssistant({
+    notifyOnChangeProps: ['data'],
+  });
+  const {mutate: mutateAssistant} = useMutateAssistant();
+
+  const shouldShowTourModal =
+    assistantData?.find(item => item.guide === EXPLORE_SPANS_TOUR_GUIDE_KEY)?.seen ===
+    false;
+
+  const isNavTourActive = useIsNavTourActive();
```

Comment:
Hmm I think this works well for now, but it does make me think that it might make sense long term to add some top-level context to ensure that only 1 tour is active at once.

Another possibility might be to prevent modals from opening if a tour is active since those don't seem to play well together.

Reply:
Yeah this is more meant to be a short term patch. A top level context/store that ensures only 1 guide is ever open.

Additionally, it might be good to prevent modals like you said during the tour. Because that's actually what the issue was that prevented the click events from responding correctly.